### PR TITLE
Content edit to Uplink article plus tidying

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: docs/source/building-supergraphs/router.mdx [hunk 1]
```diff
@@ -3,19 +3,28 @@ title: The router
 description: The entry point to your supergraph
 ---
 
-import FederationArchitecture from '../../shared/diagrams/federation-architecture.mdx';
-
 After you set up at least one federation-ready [subgraph](./subgraphs-overview/), you can configure a **router** (formerly known as a gateway) to sit in front of your subgraphs. The router serves as the entry point to your supergraph, and it executes incoming operations across one or more of your subgraphs:
 
-<FederationArchitecture />
```
Comment (StephenBarlow) [inline docs/source/building-supergraphs/router.mdx:10]: This was the only remaining use of this shared diagram. I might _re_-factor-out a diagram in the future but for now just replacing in-line

File: docs/source/building-supergraphs/subgraphs-overview.mdx [hunk 2]
```diff
@@ -66,23 +68,6 @@ type Query {
 }
 ```
 
-### `Query._service`
```
Comment (StephenBarlow) [inline docs/source/building-supergraphs/subgraphs-overview.mdx:69]: Just moving this down for alphabetization / importance relative to other field

File: docs/source/config.json [hunk 3]
```diff
@@ -21,9 +21,9 @@
       "4️⃣ Working with subgraphs": "/quickstart/local-subgraphs"
     },
     "Moving from Federation 1": {
-      "☀️ Changes from Federation 1": "/federation-2/new-in-federation-2",
```
Comment (StephenBarlow) [inline docs/source/config.json:24]: Now that this category is less prominent, scrapping the colorful icons

File: docs/source/entities-advanced.mdx [hunk 4]
```diff
@@ -101,55 +101,6 @@ type Bill @key(fields: "id") {
 
 The steps for this migration depend on how you perform schema composition:
 
-<ExpansionPanel title="With Rover CLI composition">
```
Comment (StephenBarlow) [inline docs/source/entities-advanced.mdx:104]: Moving this down below the managed federation option

File: docs/source/managed-federation/uplink.mdx
Comment (StephenBarlow) [inline docs/source/managed-federation/uplink.mdx:1]: Updated this article to cover both Router and Gateway

**Review by dariuszkuc — APPROVED** [review]

File: docs/source/entities-advanced.mdx [hunk 5]
```diff
@@ -270,9 +221,58 @@ We're done! `Bill` now originates in a new subgraph, and it was resolvable durin
...
+1. In the Billing subgraph, define the `Bill` entity exactly as it's defined in the Payments subgraph:
+
+    ```graphql title="Billing subgraph"
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+3. In the Billing subgraph, define resolvers for each field of `Bill` that are currently resolved in the Payments subgraph. These resolvers should behave identically to the resolvers in the Payments subgraph.
```
Comment (trevor-scheer) [inline docs/source/entities-advanced.mdx:235]: Should be a 2 here but also it look like MD doesn't care and it's correct on the docs site as-is anyway. Don't care if you bother to change it or not.

**Review by trevor-scheer — APPROVED** [review]

File: docs/source/entities-advanced.mdx (hunk 5 above)
Comment (StephenBarlow) [reply docs/source/entities-advanced.mdx]: Unacceptable!!! I've fixed this _glaring_ bug

**Pull request merged**